pytket/pytket/utils/term_sequence.py

```diff
@@ -28,8 +28,20 @@ def gen_term_sequence_circuit(
     partition_strat: PauliPartitionStrat = PauliPartitionStrat.CommutingSets,
     colour_method: GraphColourMethod = GraphColourMethod.Lazy,
 ) -> Circuit:
-    """Sequences QubitPauliOperator terms to generate a circuit made of CircBoxes.
-    Each CircBox contains a sequence of PauliExpBox objects.
+    """
+    Sequences a the terms of a QubitPauliOperator P in order to generate a circuit
+    approximating :math:`e^{\\frac{\\pi}{2}\\theta P}`. This method performs
+    Trotterisation with a single Trotter step.
+
+    The sequencing is done according so some specified partitioning strategy and
+    graph colouring method.
+
+    The resulting Circuit will contain a sequence of CircBoxes. Each CircBox
+    corresponds to a set of Pauli strings. Each exponentiated Pauli string
+    in the set is realised as a PauliExpBox.
+
+    The ordering of terms prioritises reducing the two qubit gate count of the
+    circuit rather than minimising trotter error.
```

Comment:
Reduces two qubit gate count _when considering applying PauliSimp or GuidedPauliSimp_. It need not minimise them for other compilation strategies.